packages/combo-box/test/lazy-loading.test.js

```diff
@@ -888,11 +877,17 @@ describe('lazy loading', () => {
 
       describe('using data provider, lost focus before data is returned', () => {
         let returnedItems;
+
         const bluringDataProvider = (params, callback) => {
           comboBox.blur();
           callback(returnedItems, returnedItems.length);
         };
 
+        const setFilterValue = (filterValue) => {
+          comboBox._inputElementValue = filterValue;
```

Comment:
Perhaps, `comboBox.value` could be used instead? 🤔 If it is set before `filter`, then observers will very likely be invoked in the correct order.

Reply:
Updated to use `comboBox.inputElement.value` instead. The reason for using `_inputElementValue` could be the fact that we used to support `iron-input` which has other property than `value` (there it was called `bindValue`).